### PR TITLE
persistent connections working again

### PR DESCRIPTION
**Pull request by prgTW**
Persistent connections are available again by not overriding PDO statement class

fixes this error:
```
Warning: PDO::setAttribute(): SQLSTATE[HY000]: General error: PDO::ATTR_STATEMENT_CLASS cannot be used with persistent PDO instances
```

ref: propelorm/Propel2#1225

### CODE REVIEW

**Comment by marcj** [comment]
We should rather completely drop Propel\Runtime\Adapter\Pdo\PdoStatement instead of using it only for HHVM.

**Comment by staabm** [comment]
alternatively we should drop HHVM support. not sure this runtime is relevant for somebody.

**Comment by marcj** [comment]
or both :P

**Comment by prgTW** [comment]
With this PR I wanted only to make persistent connections available again and didn't want to alter HHVM in any way cause I don't use it and don't have it and didn't want to make any backward-incompatible changes. Droping HHVM support could be a separate PR though

**Comment by prgTW** [comment]
so ... any other thoughts about merging this one and restoring persistent connections in propel?

**Comment by marcj** [comment]
What do you mean by dropping hhvm support? This drops basically the overwritten class support, for whatever that was used. Let us just fix it the right way and remove `Propel\Runtime\Adapter\Pdo\PdoStatement` completely, instead of using it only for hhvm.

**Comment by prgTW** [comment]
@marcj I removed `Propel\Runtime\Adapter\Pdo\PdoStatement` class as You suggested

File: src/Propel/Runtime/Adapter/Pdo/PdoStatement.php [hunk 1]
```diff
@@ -1,101 +0,0 @@
...
- * This file is part of the Propel package.
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- *
- * @license MIT License
- */
-
-namespace Propel\Runtime\Adapter\Pdo;
-
-use Propel\Runtime\Connection\StatementInterface;
```
Comment (prgTW) [inline src/Propel/Runtime/Adapter/Pdo/PdoStatement.php]: actually it's used in quite few places, shouldn't it's usages be converted to usages of `\PdoStatement` as well, am I getting You right?

Comment (prgTW) [inline src/Propel/Runtime/Adapter/Pdo/PdoStatement.php]: removed

**Comment by marcj** [comment]
Nice, good work, thanks!

**Comment by prgTW** [comment]
great news, thanks you too :)

**Comment by marcj** [comment]
Backported to data-mapper branch.

**Comment by denormalizer** [comment]
Hi guys. 

This pull request broke something that used to work: #1413

In composer.json, I've currently had to make sure propel use the commit prior to this:

    "propel/propel": "dev-master#38f1044e9b57cb7f68b59224f1830bde7009673f"

Does anyone have a workaround so I don't have to be stuck on this commit?

**Pull request merged**